backend/app/deployment_handler.go

```diff
@@ -316,7 +317,11 @@ func (h *Handler) HandleDeployCluster(c *gin.Context) {
 		"cluster": cluster,
 	}
 
-	h.ewfEngine.RunAsync(h.appContext, wf)
+	if err = h.ewfEngine.RunAsync(h.appContext, wf, ewf.WithQueue(strconv.Itoa(config.UserID))); err != nil {
```

Comment:
I assume this relies on the default configuration for the queue which can change or just not match what we need. If we have a specific constraints about the queue config we should probably configure the queue first with the requirements (so 1 worker, auto delete, etc...)

Reply:
also I would make the queue name more descriptive to imply that this queue is for chain operations in case we decide to add another queue for the user